CMakeLists.txt

```diff
@@ -918,10 +918,10 @@ IF (WITH_H2OLOG)
 
     ADD_EXECUTABLE(h2olog ${H2OLOG_SOURCE_FILES})
 
-    SET_PROPERTY(TARGET h2olog
-        PROPERTY CXX_STANDARD 11
-        PROPERTY CXX_STANDARD_REQUIRED ON
-        PROPERTY CXX_EXTENSIONS OFF
+    SET_TARGET_PROPERTIES(h2olog PROPERTIES
```

Comment:
Oh, I thought `SET_TARGET_PROPERTIES` was not available on cmake 2.8.11.

Seems better than https://github.com/h2o/h2o/pull/2367